pkg/job_controller/api/v1/constants.go

```diff
@@ -45,6 +48,16 @@ const (
 	ResourceNvidiaGPU v1.ResourceName = "nvidia.com/gpu"
 )
 
+// add annonations
+const (
+	AnnotationPodBatchSize        = KubeDLPrefix + "/pod-batch-size"
+	AnnotationPodAffinityStrategy = KubeDLPrefix + "/pod-affinity-strategy"
+	AnnotationPodTopologyKey      = KubeDLPrefix + "/pod-topology-key"
```

Comment:
IMO, `pod-topology-key` should not be specified by users, it can be defaulted as `job-name` label since all pods will be labeled by job name